crates/swc_plugin_runner/benches/invoke.rs

```diff
@@ -0,0 +1,82 @@
+extern crate swc_node_base;
+
+use std::{
+    env,
+    path::{Path, PathBuf},
+    process::Command,
+    sync::Arc,
+};
+
+use criterion::{black_box, criterion_group, criterion_main, Bencher, Criterion};
+use once_cell::sync::Lazy;
+use swc_common::{plugin::Serialized, FileName, FilePathMapping, SourceMap};
+use swc_ecma_ast::EsVersion;
+use swc_ecma_parser::parse_file_as_program;
+use swc_plugin_runner::cache::PluginModuleCache;
+
+static SOURCE: &str = include_str!("./assets/input.js");
+
+fn plugin_group(c: &mut Criterion) {
+    let plugin_dir = PathBuf::from(env::var("CARGO_MANIFEST_DIR").unwrap())
+        .join("..")
+        .join("..")
+        .join("tests")
+        .join("rust-plugins")
+        .join("swc_internal_plugin");
+
+    {
+        let mut cmd = Command::new("cargo");
+
+        cmd.current_dir(&plugin_dir);
+        cmd.arg("build")
+            .arg("--release")
+            .arg("--target=wasm32-unknown-unknown");
```

Comment:
one day we may compare wasm32-wasi as well maybe?

Reply:
Yeah I think we should